pkgs/development/ocaml-modules/hxd/default.nix

```diff
@@ -5,22 +5,17 @@
 
```

Comment:
I think that `withLwt` and `ocaml` are no longer necessary here.

Reply:
right, it's still optional, but we can just default to `true`.